packages/sanity/src/structure/panes/documentList/SheetListCell.tsx

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable i18next/no-literal-string */
+import {Select, TextInput} from '@sanity/ui'
+import {type CellContext} from '@tanstack/react-table'
+import {type FormEvent, useCallback, useEffect, useState} from 'react'
+import {type SanityDocument} from 'sanity'
+
+interface SheetListCellProps extends CellContext<SanityDocument, unknown> {
+  type: any
+}
+
+export function SheetListCell(props: SheetListCellProps) {
```

Comment:
This is pretty much just a re-hash of what has now been removed from `useDocumentSheetColumns.ts`, so that it forms it's own component